src/i18n/faucet_fr.json

```
@@ -0,0 +1,24 @@
+{
+  "faucet": {
+    "common": {
+      "error": "Erreur",
+      "success": "Succès"
+    },
+    "sendMeToken": "Envoyez-moi des AXONE",
+    "requestFundsWithKeplr": "Demander des fonds avec Keplr",
+    "requestFundsWithAddress": "Demander des fonds avec une adresse de la chaine AXONE",
+    "addressPlaceholder": "axonexxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "operationSucceeded": "Votre transaction a été envoyée avec succès!",
+    "brand": {
+      "title": "AXONE Faucet",
+      "description": "Bonjour camarade! Utilisez ce faucet afin d'obtenir des jetons pour la chaîne {{env}}. Veuillez ne pas abuser de ce service - le nombre de jetons disponibles est limité."
```

@ccamel This in french is OK for you ?

```suggestion
      "description": "Salutations, explorateur web3 ! Réclamez votre jeton AXONE sur la chaîne {{env}} pour lancer votre voyage dans l'écosystème Axone. N'oubliez pas qu'il s'agit d'une ressource partagée - la disponibilité des jetons est limitée."
```

---

@amimart ok for you?